faf-commons-api/src/main/java/com/faforever/commons/api/dto/Player.java

```diff
@@ -39,6 +40,12 @@
     @JsonIgnore
     private List<AvatarAssignment> avatarAssignments;
 
+    @Relationship("reporterOnModerationReports")
+    private Set<ModerationReport> reporterOnModerationReports;
```

Comment:
Same as above. But you will notice as soons as you modify modor. You can change it then I guess. Btw you can also use none develop commits to use modor with, you dont need to wait for this getting merged into develop ;)